src/grafana/provisioning/dashboards/demo/demo-dashboard.json

```diff
@@ -1039,6 +1039,6 @@
   "timezone": "",
   "title": "Demo Dashboard",
   "uid": "W2gX2zHVk",
-  "version": 2,
+  "version": 1,
```

Comment:
any reason why you are setting version to 1?

Reply:
I didn't set it this specifically. The Grafana UI spit out the dashboard JSON.